### DISCUSSION TITLE
UV-K5、K6 has a background noise on the frequency 130.00mhz，even without the antenna。

### DISCUSSION THREAD

**Discussion by cnbi4oqo** [egzumer/uv-k5-firmware-custom#419] [Q&A]
UV-K5、K6  has a 130.00mhz  noise ，even without the antenna。
Request a update to solve the problem。

73 to all  
    
               a ham from china

**Comment by OE1MWW** [comment]
UV-K5(8) same 0.21 firmware on each unit
device (1) produced 10/2023: S4 to S5 on the S-meter
device (2) produced 11/2023: S2 on the S-meter

so, I guess, it's from production / hardware

73s

**Reply by cnbi4oqo** [reply]
thx for your reply！
i agree with your opinion，just pull this problem to the hams and try to get a solution from A MASTER。
73
![微信图片_20240116112659](https://github.com/egzumer/uv-k5-firmware-custom/assets/156640062/a4e95602-83c0-468f-a7df-6c3f20cabf58)
![微信图片_20240116112659](https://github.com/egzumer/uv-k5-firmware-custom/assets/156640062/5acc1801-309d-4942-8ce9-6bc7521be3d6)
！

**Comment by iu0azb** [comment]
Hi,
Please look into this discussion:

https://github.com/egzumer/uv-k5-firmware-custom/discussions/323

Only solution to mitigate the problem was to shield the xtal area.

**Reply by cnbi4oqo** [reply]
thx for your reply！
but i have no capacity to do it by myself，i am a ham without any  hands-on ability。
by the way，the callsign with “iu0” is belongs to which nationality？

73s from china~~

**Comment by OE1MWW** [comment]
as said - the 'noise' you hear is from your own hardware. What could 'A MASTER' do ?
A blind spot in the firmware for each and every 26MHZ multiple ???
See [#323](https://github.com/egzumer/uv-k5-firmware-custom/discussions/323)

> by the way，the callsign with “iu0” is belongs to which nationality？

that's easy: https://www.qrz.com/db/IU0IJV will give you all details

来自奥地利的 73s

**Comment by OE1MWW** [comment]
sorry Luca, you are on the QRZ.com list too ;-) [IU0AZB]([iu0azb](https://github.com/iu0azb))

**Comment by BruMcD** [comment]
It is the 5th Harmonic of a 26MHz crystal oscillator inside the set.  The only software 'fix' would be to reduce the RF gain on that frequency
